nucypher/network/nodes.py

```diff
@@ -390,51 +501,21 @@ def remember_node(self,
                 # This node is already known.  We can safely return.
                 return False
 
-        self.known_nodes[node.checksum_address] = node
-
-        if self.save_metadata:
-            self.node_storage.store_node_metadata(node=node)
+        self._remember_essential(node=node)
+        if not node.verified_node:
+            # set UNVERIFED bucket as default, but don't relabel
+            # already verified nodes as unverified.
+            self.known_nodes.label(node=node, label=UNVERIFIED)
```

Comment:
Note to self: this logic is incorrect, instead check for the existence of any label, don't force `UNVERIFED` implicitly.

Reply:
✔️